plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/ILibraryRepositoryService.java

```diff
@@ -18,29 +18,32 @@
 
 import com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFile;
 import com.google.cloud.tools.eclipse.appengine.libraries.model.MavenCoordinates;
-import org.eclipse.jdt.core.IClasspathEntry;
+import org.apache.maven.artifact.Artifact;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IProgressMonitor;
 
 /**
  * Service interface for obtaining local paths for artifacts described by {@link MavenCoordinates}
  */
 public interface ILibraryRepositoryService {
```

Comment:
Does it need to be a service?  It's only used by `LibraryClasspathContainerResolverService`, which could create it directly.  The other references I see , in `ServletClasspathProvider*`, are not actually used.